guess-the-country/country-frontend/src/App.vue

```diff
@@ -35,41 +34,49 @@
           v-bind:src="this.streetviewimage"
         />
       </section>
-
       <Selection
         @city_selected="city_selected"
         :label_city="label_city"
         :user_city_answer="user_city_answer"
+        :sequence_mode="sequence_mode"
+        :prediction_city="prediction_city"
+        :explanation="explanation"
       />
-
-      <button
-        v-if="!prediction_city && user_city_answer && !control && round != 11"
+      <!-- what do you guess, AI Button - treatment group -->
+      <button  
+        v-if="sequence_mode==='classic'&&!prediction_city && user_city_answer && !control && round != 11
+              ||sequence_mode==='recommender'&& !prediction_city && !control && round != 11" 
         type="button"
         class="xd-button xd-secondary"
         id="explain"
         v-on:click="explain()"
       >
-        <!-- v-show -->
         What do you guess, AI?
       </button>
+      <!-- next button --> 
       <button
-        v-if="(explanation || (control && prediction_city)) && round < 11"
+        v-if="sequence_mode==='classic'&&(explanation || (control && prediction_city)) && round < 11
+              || sequence_mode==='recommender' && user_city_answer && round < 11
+              || sequence_mode==='basic' && user_city_answer"
```

Comment:
Ich fände es gut, wenn wir diese komplexen Prüfungen in eine Computed-Methode auslagern können.

Dann steht hier nur: 

```html
<button v-if="nextButtonVisible" ...
```

und unten

```javascript
computed: {
    nextButtonVisible() {
        return this.sequence_mode === 'classic' ...
}
}
```

Die Prüfung kann man dann vielleicht auch als einfacher nachzuvollziehende `if`-`else`-Konstruktion gestalten.

Reply:
Soll es für jede div/section/button/... eine eigene cumputed-Methode geben, in welcher die Logik drin steckt?

Reply:
Nein, da reicht eine, oder? Die spezifischen Abfragen kann man dann ja anhängen